code/game/objects/structures/window_frame.dm

```diff
@@ -9,19 +9,16 @@
 	throwpass = TRUE
 	climbable = 1 //Small enough to vault over, but you do need to vault over it
 	climb_delay = 15 //One second and a half, gotta vault fast
+	smoothing_behavior = CARDINAL_SMOOTHING
+	smoothing_groups = SMOOTH_GENERAL_STRUCTURES
 	var/obj/item/stack/sheet/sheet_type = /obj/item/stack/sheet/glass/reinforced
 	var/obj/structure/window/framed/mainship/window_type = /obj/structure/window/framed/mainship
 	var/basestate = "window"
 	var/junction = 0
 	var/reinforced = FALSE
 
-	tiles_with = list(
-		/turf/closed/wall)
+	smoothing_groups = SMOOTH_GENERAL_STRUCTURES
```

Comment:
```suggestion
```

defined twice

Reply:
This is a depart from the original system and would make them not smooth with anything, not even themselves, no?

Reply:
No i literally just meant its duplicated from line 13.
![image](https://user-images.githubusercontent.com/1134201/87467815-6e44cb00-c610-11ea-9f96-982c3d50f3ef.png)